pythreejs/traits.py

```diff
@@ -154,15 +157,25 @@ class WebGLDataUnion(DataUnion):
     Also constrains the use of 64-bit arrays, as this is not supported by WebGL.
     """
     def validate(self, obj, value):
+        was_original_array = isinstance(value, np.ndarray)
+        print(value, was_original_array)
         value = super(WebGLDataUnion, self).validate(obj, value)
         array = value.array if isinstance(value, NDArrayWidget) else value
 
-        if array is not Undefined and str(array.dtype) == 'float64':
+        dtype_str = str(array.dtype) if array is not Undefined else ''
+        if dtype_str == 'float64' or dtype_str.endswith('int64'):
```

Comment:
float64 is supported, just not (u)int64 right?

Reply:
Ah wait.. WebGL doesn't support float64, forget about that.

Reply:
Javascript doesn't support Float64 arrays. WebGL adds the (u)int restraint for textures/buffers.

Reply:
I mean the opposite 😅

Reply:
👍